testcontainers/Cargo.toml

```diff
@@ -16,6 +16,8 @@ all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
+async-std = "1.13.0"
+async-tar = "0.5.0"
```

Comment:
Please, take a look at https://github.com/testcontainers/testcontainers-rs/pull/730#discussion_r1752678907

We can use tokio-tar

Reply:
Oh sorry. I overlooked that. yes sure. I'll change that

Reply:
Done